conf/application.conf

```diff
@@ -120,7 +120,7 @@ assets {
 }
 
 contact-frontend {
-  host = "http://localhost:9250"
+  url = "http://localhost:9250"
```

Comment:
If you change this you will need to also create an app-config-base PR and update the variable name there